README.md

```diff
@@ -1,24 +1,50 @@
-# README
+# Daily Expenses
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Daily Expenses is a budget management application where users can register and use the features of the application. Users can create categories, assign the categories a particular icon, and add expenses to the categories. Users can also see the total amount spent on each category.
 
-Things you may want to cover:
+![screenshot](./gif.gif)
 
-* Ruby version
```

Comment:
- [ ] Kindly deploy your application and attached the **_Live version_** Link in your `README.md` file. Please ensure that the project works properly for all pages on the deployed version. 👍